packages/relay/tests/lib/eth.spec.ts

```diff
@@ -893,6 +893,61 @@ describe('Eth calls using MirrorNode', async function () {
     expect(result).to.equal(null);
   });
 
+  describe('eth_getBalance', async function() {
+    it('should return cached value', async () => {
+      mock.onGet(`blocks?limit=1&order=desc`).reply(200, {
+        blocks: [{
+          number: 10000
+        }]
+      });
+      mock.onGet(`contracts/0x000000000000000000000000000000000000055f`).reply(200, null);
+      mock.onGet(`accounts/0x000000000000000000000000000000000000055f`).reply(200, {
+        account: '0x000000000000000000000000000000000000055f'
+      })
+      sdkClientStub.getAccountBalanceInWeiBar.throws({
```

Comment:
Can you add a check that ensures the stub for `getAccountBalanceInWeiBar` is only called once.
This verifies the cache was picked up